internal_docs/releasing.md

```diff
@@ -44,7 +44,7 @@ v8.0.0 // After implementing backwards incompatible changes
 
 9. In Tag version and Release title, put v[version number], for example `v7.0.0`.
 
-10. In description, paste the relevant section from the Changelog.
+10. In description, paste the relevant section from the release notes in the Google Doc.
```

Comment:
Is there a step before this about asking a content designer to make the release notes in a Google Doc?

Reply:
Fair point. Anyone can draft release notes - and often it's developers who do a first cut. 
Could there be a step around 'Draft release notes in a Google Doc' without saying who?

Reply:
Sounds good to me!